include/SFML/Graphics/PrimitiveType.hpp

```diff
@@ -36,7 +36,7 @@ namespace sf
 /// and view.
 ///
 ////////////////////////////////////////////////////////////
-enum PrimitiveType
+enum class PrimitiveType
```

Comment:
I thought there was talk about using enum struct over enum class?

Reply:
As far as I know there's no difference, but `enum class` is the de facto standard

Reply:
`enum class` is what SFML is already using so let's please keep using that.